WordPress/src/main/java/org/wordpress/android/ui/main/SitePickerAdapter.java

```diff
@@ -472,7 +472,7 @@ private SiteList filteredSitesByText(SiteList sites) {
      * AsyncTask which loads sites from database and populates the adapter
      */
     private boolean mIsTaskRunning;
```

Comment:
I know it's not part of your changes, but what do you think of removing this flag? as it's not used